Dockerfile

```diff
@@ -9,7 +9,8 @@ RUN apt-get update && \
 COPY ./requirements.txt /app/requirements.txt
 
 RUN pip install -U 'pip>=8' && \
-    pip install --no-cache-dir -r requirements.txt
+    pip install --no-cache-dir -r requirements.txt\
+    pip install newrelic
```

Comment:
This is missing an `&&`, so it's concatenating this line and the next as a single "pip install" and there's no package named "install".

Further, we really want to pin newrelic and use a hash, so I'm going to move that to a `requirements_newrelic.txt` file.